Tests/PerformanceTests.swift

```diff
@@ -34,7 +34,7 @@ class PerformanceTestCase: BaseTestCase {
         let dataArray: [AnyObject] = {
             var array: [AnyObject] = []
             for _ in 0...1000 {
-                array.append(json.copy())
+                array.append(json as AnyObject)
             }
             return array
```

Comment:
Rather than typecast here, you could do the following:

```swift
var array: [Any] = []

for _ in 0...1000 {
   array.append(json)
}
```

Then you wouldn't need a type cast.